exporter/tanzuobservabilityexporter/README.md

```diff
@@ -1,21 +1,30 @@
 # Tanzu Observability (Wavefront) Exporter
 
-This exporter supports sending traces to [Tanzu Observability](https://tanzu.vmware.com/observability).
+This exporter supports sending both traces and metrics to [Tanzu Observability](https://tanzu.vmware.com/observability).
 
 ## Prerequisites
 
 - [Obtain the Tanzu Observability by Wavefront API token.](https://docs.wavefront.com/wavefront_api.html#generating-an-api-token)
 - [Set up and start a Tanzu Observability by Wavefront proxy](https://docs.wavefront.com/proxies_installing.html) and configure it with the API token you obtained.
 - To have the proxy generate [span RED metrics](https://docs.wavefront.com/trace_data_details.html#red-metrics) from trace data, [configure](https://docs.wavefront.com/proxies_configuring.html) the proxy's `customTracingListenerPorts` and use it for the exporter's endpoint.
 
-## Data Conversion
+## Data Conversion for Traces
 
 - Trace IDs and Span IDs are converted to UUIDs. For example, span IDs are left-padded with zeros to fit the correct size.
 - Events are converted to [Span Logs](https://docs.wavefront.com/trace_data_details.html#span-logs).
 - Kind is converted to the `span.kind` tag.
 - Status is converted to `error`, `status.code` and `status.message` tags.
 - TraceState is converted to the `w3c.tracestate` tag.
 
+## Data Conversion for Metrics
+- Gauge metrics are converted to Gauge metrics
+- Cumulative sum metrics are converted to counter metrics (not sure if there is any difference between a Wavefront Gauge metric and a Wavefront Counter metric)
+- Delta sum metrics are converted to delta counters
+- Each cumulative histogram metric is converted to multiple counter metrics (one counter metric per bucket in the histogram). Each counter metric has a special "le" tag that matches the upper bound of the corresponding bucket. The count in the counter metric is the sum of the corresponding bucket and all the buckets before it.
+- Delta histograms get converted to histograms in wavefront
+- Exponential histograms are converted the same way as regular histograms
```

Comment:
If there's no information loss or data shape change, I don't see a need to list every type. @oppegard what do you think?

Reply:
I agree, but I also don't think it hurts to make it plainly obvious what the mapping is. Perhaps bullets one, two, and four can be collapsed into a single item like:

- OpenTelemetry Gauges, Sums, and Delta Histograms convert to [Wavefront Gauges, Counters, and Histograms](https://docs.wavefront.com/metric_types.html), respectively.

Other nit: let's end each complete sentence in a bullet point with a period.

Reply:
We made a chart. with some notes below.